packages/web-client/src/helpers/resource/functions.ts

```diff
@@ -99,6 +99,7 @@ export function buildResource(resource): Resource {
     webDavPath: resource.name,
     type: isFolder ? 'folder' : resource.type,
     isFolder,
+    processing: resource.processing || false,
```

Comment:
!!resource.processing

Reply:
Why? It's a boolean in the response. The `owncloud-sdk` only ever sets `true` or `false` as values.